uiowa/uiowa
# Started adding area of study lb layout

Closes #2815 


# How to test


1. blt ds --site=admissions.uiowa.edu
2. Go to the [area of study CT default display](http://admissions.local.drupal.uiowa.edu/admin/structure/types/manage/area_of_study/display/default), select 'Manage Layout' and make sure all fields are visible/available in the two column display
